pkgs/development/libraries/libadwaita/default.nix

```diff
@@ -45,6 +46,7 @@ stdenv.mkDerivation rec {
     sassc
     vala
     gobject-introspection
+    appstream
```

Comment:
This should go to `buildInputs` since it is meant to be used at runtime for about dialog. But we will need to check what effect this has on closure size and possibly try to make a slimmer libappstream package.